### ISSUE TITLE
Disallow investor rondel if dividends can't be paid

### ISSUE THREAD

**Issue by dmcarmo** [Thrillberg/imperial#309]
If a nation doesn't have enough funds to pay all its dividends. the controlling player **may** choose to not get payed in order to fulfill this action. If it's still not possible to pay the remaining investors them the controlling player **must** spend from his cash to fulfill this action.
If both the nation and the controlling player don't have enough funds to pay the other investors dividends, then the Investor slot should be disallowed.

**Comment by Thrillberg** [comment]
Can a Swiss Bank force the player to land on the Investor slot if they cannot afford to pay all interest (including the nation's funds)?

**Comment by PedroRCSequeira** [comment]
Nope, it cant be forced unless everything can be paid (edit: paid fully from the nations treasury)

**Comment by Thrillberg** [comment]
Do we know that the Investor slot is disallowed in this case? I just read my rules book and it seems ambiguous about whether the player should make up as much as they can (without going into negative cash, presumably) or if they just cannot land there at all. Did Mac clarify this rule somewhere else or anything?

I'm ok going either way with it, I just want to be sure we're keeping as close to the rules as possible.

**Comment by PedroRCSequeira** [comment]
You're asking about going into investor, and from nations treasury + controlling player still not being able to pay the other investors?

I would guess it is not allowed, i dont think there is any mention in the rules for what would happen (if only a few would be paid, or if payment would come from the "main bank", etc

But most importantly, I'm pretty sure it would never happen intentionally ^^
(even rarer than the "funding nations that you dont control" eheh

**Comment by Thrillberg** [comment]
> I'm pretty sure it would never happen intentionally

That's great to know and I agree! Maybe this should _also_ be de-prioritized.

**Comment by dmcarmo** [comment]
Agreed, it's something that should be implemented but there are more urgent issues.
